aux/files/js/script.js

```diff
@@ -61,17 +61,13 @@ $(document).ready(function() {
 });
 
 $(function(){
-
-  function toggleRow(){
-    if ($(this).hasClass('success') || $(this).hasClass('danger')){
-      var title = $(this).attr('title');
-      var val = title.replace(/[ \/]/g, '');  
-      addData(this, val);
-    } else if ($(this).hasClass('plot_btn')){
-      addData(this, 'all');
-    }
-  }
-
-  $("td, .plot_btn").click(toggleRow);
-
+  $( "#sortable_table" ).on( "click", "td, .plot_btn", function( event ) {
```

Comment:
I would write it as `$(document).on('click', '.plot_btn', ...)`. For one, `sortable_table` doesn't feel like a very stable name. Second, events can be attached to `document` even before `onready` has fired. Finally, if we move to full client side rendering the table may not exist when the event handler is attached.